src/duckling/game/drawable/container-drawable.component.ts

```diff
@@ -57,6 +59,17 @@ export class ContainerDrawableComponent {
     @Input() containerDrawable : ContainerDrawable;
     @Output() drawableChanged = new EventEmitter<ContainerDrawable>();
 
+    onChildDrawableCloned(newDrawables : Drawable[]) {
```

Comment:
Any suggestions? The majority of the work is generating the new unique key, but that's dependent on the content of the accordion.